Arrowgene.Ddon.Shared/Entity/PacketStructure/C2SQuestQuestLogInfoRes.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Entity.Structure;
+using Arrowgene.Ddon.Shared.Model;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure
+{
+    public class C2SQuestQuestLogInfoRes : ServerResponse
```

Comment:
Shouldn't this class be S2CQuestQuestLogInfoRes instead?

Reply:
Yes. Done.